### PR TITLE
Geojs component

### PR DESCRIPTION
**Pull request by ronichoudhury**
Closing #171 in favor of this PR.

### CODE REVIEW

**Comment by ronichoudhury** [comment]
FYI, I rebased this against master (mainly to get proper Travis CI tests going).

**Comment by ronichoudhury** [comment]
@aashish, I pushed a few changes here, mainly to deal with linting/style issues, but also a few things of substance (e.g., using GeoJS 0.9.1 now to solve the webpack-related issues).

File: package.json [hunk 1]
```diff
@@ -99,6 +99,8 @@
     "font-awesome-webpack": "^0.0.4",
     "jquery": "^2.2.0",
     "jquery-deparam": "^0.5.2",
+    "LineUpJS": "git+https://github.com/Caleydo/lineup.js#51ae6f2bcb55aead3f63357864e90d5799789910",
```
Comment (jeffbaumes) [inline package.json]: There are two LineUpJS's now.

Comment (jeffbaumes) [inline package.json]: Oops looks like I needed to refresh.

**Comment by jeffbaumes** [comment]
@ronichoudhury @aashish24 What's the next step on this PR?

**Comment by aashish24** [comment]
@jeffbaumes we will post the step this afternoon but in short 1) fix/update data spec 2) support for line and polygons 3) testing 4) componentize

**Comment by ronichoudhury** [comment]
I think 3 from Aashish's list is all that's necessary to merge this PR; the rest we can do as future topics. But we've got a meeting this afternoon to talk about/do some of these things.

**Comment by ronichoudhury** [comment]
I believe this is ready for review. @aashish24, please be aware that I rebased this against current master again, so your remotes are likely out of sync with origin.

**Comment by aashish24** [comment]
Roger that @ronichoudhury

**Comment by ronichoudhury** [comment]
@aashish24, @jeffbaumes, @alex-r-bigelow: I'm closing this PR in favor of #257 (which is based on this branch in any case) because of the morass of testing fixes I've had to make which affect both PRs but will be annoying to backport to this branch. Ordinarily I'd make the effort to keep the PRs relatively independent but since the Geo and GeoDots components are intimately related anyway, I think it will be ok this time.

**Pull request closed without merging**